tests/test_io.py

```diff
@@ -387,12 +387,12 @@ def test_json():
     check(assert_type(read_json(bin_json), DataFrame), DataFrame)
 
 
-def test_json_series():
+@pytest.mark.parametrize("orient", [None, "split", "records", "index", "table"])
+def test_json_series(orient):
```

Comment:
I think you would need to type this parameter so that mypy/pyright can test the stubs

Reply:
As in `012a9b8`? Otherwise I'll more input.